File: app/src/main/java/org/fossasia/susi/ai/rest/model/Identity.java
package org.fossasia.susi.ai.rest.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Identity {

    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("type")
    @Expose
    String type;
    @SerializedName("anonymous")
    @Expose
    Boolean anonymous;

    public String getType() {
        return type;
    }

    public String getName() {
        return name;
    }

    public Boolean getAnonymous() {
        return anonymous;
    }
}

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Identity.kt
package org.fossasia.susi.ai.rest.responses.susi

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Identity(
    val name: String? = null,
    val type: String? = null,
    val anonymous: Boolean = false
) : Parcelable
